config/environment.rb

```diff
@@ -3,3 +3,13 @@
 
 # Initialize the Rails application.
 Rails.application.initialize!
+
+ActionMailer::Base.smtp_settings = {
+  user_name: Rails.application.secrets.sendgrid_username,
+  password: Rails.application.secrets.sendgrid_password,
+  domain: 'getguesstimate.com',
```

Comment:
what do you mean getting repeated everywhere? I don't see many places where it is repeated natively atm.